### PR TITLE
feat: nano version of lru cache without additional code and with mini…

### PR DESCRIPTION
**Pull request by Tom910**
…mal changes

### CODE REVIEW

**Comment by Tom910** [comment]
As a result
Before - Size: 3.58 kB
This mr - Size: 1.29 kB

File: index.js [hunk 1]
```diff
@@ -227,17 +103,6 @@ class LRUCache {
     this.setItemTTL = (index, ttl) => {
       this.starts[index] = ttl !== 0 ? perf.now() : 0
       this.ttls[index] = ttl
-      if (ttl !== 0 && this.ttlAutopurge) {
```
Comment (SuperOleg39) [inline index.js:230]: А это разве не важный функционал? Или ttlAutopurge это экзотический кейс?

**Review by SuperOleg39 — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (Tom910) [reply index.js:230]: ага, у нас не используется и влияло на производительность

**Comment by meskill** [comment]
types for the lru-cache come from a separate package [@types/lru-cache](https://www.npmjs.com/package/@types/lru-cache).

I think we need nano version of this as well, maybe it will be simpler to bundle it in the current package

**Comment by meskill** [comment]
types for the lru-cache come from a separate package [@types/lru-cache](https://www.npmjs.com/package/@types/lru-cache).

I think we need nano version of this as well, maybe it will be simpler to bundle it in the current package

**Pull request merged**